Runtime/Scripts/GltfImport.cs

```diff
@@ -1170,6 +1170,30 @@ public class GltfImport : IGltfReadable, IGltfBuffers {
 
 #endif // MESHOPT
 
+        private struct SamplerKey : IEquatable<SamplerKey> {
```

Comment:
Looks good, but should go into separate file.

Reply:
Not sure if this is the right place, but I moved this to glTFast\Runtime\Scripts\SamplerKey.cs